config/secrets.yaml.template

```diff
@@ -9,7 +9,7 @@ groot:
 jwt:
   secret: 'SECRET_JWT_TOKEN'
 
-development:	
```

Comment:
perhaps make it development_db and test_db since there is no filename context now